JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,31 @@
 //
 //  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
+//  Created by dasan & kyungmin.
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-// 과일 저장소 타입
 class FruitStore {
+    private(set) var fruitInventory: [FruitType: Int] = [.strawberry: 10, .banana: 10, .pineapple: 10, .kiwi: 10, .mango: 10]
+    
+    func checkStock(fruit: FruitType, amount: Int) -> Bool {
+        guard let fruitStock = fruitInventory[fruit],
+              fruitStock >= amount else {
+            return false
+        }
+        return true
+    }
+    
+    func addStock(fruit: FruitType, addCount: Int) {
```

Comment:
addStock이라는 메서드명에서 추가한다는 의미를 충분히 전달하고 있어서 매개변수 명이 addCount일 필요는 없을거 같네요!
```suggestion
    func addStock(fruit: FruitType, count: Int) {
```

API Design Guidelines의 [Promote Clear Usage](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage)를 참고해보세요!

아 그리고, 위 메서드에선 `amount`를 사용하셨는데 여기서 `count`를 사용하신 이유를 알 수 있을까요??

Reply:
재고 수량을 빼주고 더해준다는 개념에서 count를 사용했었는데 양을 빼주고 더해주는 의미로 통일해서 amout로 변경했습니다.